wled00/data/settings_wifi.htm

```diff
@@ -6,18 +6,100 @@
 	<meta content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" name="viewport"/>
 	<title>WiFi Settings</title>
 	<script>
-		var d=document;
-		var loc = false, locip;
+		let d = document;
+		let loc = false, locip;
+		let scanLoops = 0;
+
+		function Local() {
+			if (window.location.protocol == "file:") {
+				loc = true;
+				locip = localStorage.getItem("locIp");
+				if (!locip) {
+					locip = prompt("File Mode. Please enter WLED IP!");
+					localStorage.setItem("locIp", locip);
+				}
+			}
+		}
 		function H(){window.open("https://kno.wled.ge/features/settings/#wifi-settings");}
 		function B(){window.open("/settings","_self");}
+		function N() {
+			const url = (loc?`http://${locip}`:"") + "/json/networks";
+
+			const button = d.getElementById("scan");
```

Comment:
It may be a good idea to make `function gId(e) { return d.getElementById(e); }` at the beginning to save some flash memory and then use `gId("scan");`.

Reply:
Good idea, I also did the same for `function cE(e) { return d.createElement(e); }` since it was used a few times as well.